releasenotes/notes/num-ancilla-qubits-deprecation-d616c3fc24250d22.yaml

```
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    The property ``num_ancilla_qubits`` from the class :class:`~qiskit.circuit.library.arithmetic.PolynomialPauliRotations` 
+    has been removed, as deprecated in Qiskit 0.23.0.
+    Instead, use the property :attr:`~num_ancillas`.
```

I don't think sphinx will resolve this correctly. You probably need to give it a bit more context on the reference. Something like this:

```suggestion
    Instead, use the property :attr:`.PolynomialPauliRotations.num_ancillas`.
```

should work it will also make the text read `PolynomialPauliRotations.num_ancillas` instead of just `num_ancillas`